app/Content.php

```diff
@@ -17,10 +17,32 @@ class Content extends Model
 
     protected $dates = ['created_at', 'updated_at', 'start_at', 'end_at'];
 
-    protected $appends = ['body_filtered', 'image_id'];
+    protected $appends = ['body_filtered', 'image_id', 'unread', 'first_unread_comment_id'];
 
     // Relations
 
+    public function getUnreadAttribute()
```

Comment:
Why not keep it all at V2ContentVars? We moved all magical computed vars and attributes out behind presenter class, accessible via ->vars(), keeping only db-based attributes and relations in model php file. We could revisit that later (bring that stuff in via traits etc) but ->vars() is what we standardized for now.